From 1783d47ce821beb9e83723be3e9961a784a15319 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:34:52 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the direct GnuPG verification with %openpgpverify, which
uses Sequoia sqv, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Update p11-kit-release-keyring.gpg to the current copy at
https://p11-glue.github.io/p11-glue/p11-kit/p11-kit-release-keyring.gpg,
the URL the spec names for it (same certificate(s)).

Assisted-by: Claude Opus 5.5
---
 p11-kit.spec | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/p11-kit.spec b/p11-kit.spec
index bbc8f07..ede86c9 100644
--- a/p11-kit.spec
+++ b/p11-kit.spec
@@ -33,6 +33,7 @@ BuildRequires:  pkgconfig(glib-2.0)
 BuildRequires:  pkgconfig(systemd)
 BuildRequires:  gnupg2
 BuildRequires:  /usr/bin/xsltproc
+BuildRequires:  openpgpverify
 
 %if %{with mingw}
 BuildRequires:  ninja-build
@@ -137,7 +138,7 @@ way that they're discoverable.  This library is cross-compiled for MinGW.
 
 
 %prep
-gpgv2 --keyring %{SOURCE2} %{SOURCE1} %{SOURCE0}
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 
 %autosetup -p1
 
-- 
2.56.0

